tutorials/Python/Hyperspectral/hdf5-for-hyperspectral-data/Python-HDF5-BasicScript/Python-HDF5-BasicScript.md

```diff
@@ -2,9 +2,10 @@
 syncID: a9b616f171be48d7a0ea2d539de11c87
 title: "Open HDF5 files with Python Sample Code" 
 description: "This tutorial provides basic code for opening an HDF5 file in Python using the h5py, numpy, and matplotlib libraries."
-dateCreated:   2015-5-08 
+dateCreated: 2015-5-08
+dateUpdated: 2020-04-24
```

Comment:
So this is a good idea to include a note on when it was last updated, but our new website will actually have a way to scrape this information from GitHub,, so we do not need to add this to the YAML header